Backpack/Calendar/Classes/BPKCalendarStickyHeader.m

```diff
@@ -37,7 +37,7 @@ - (instancetype)initWithFrame:(CGRect)frame {
     self = [super initWithFrame:frame];
 
     if (self) {
-        self.titleLabel.textAlignment = NSTextAlignmentLeft;
+        self.titleLabel.textAlignment = NSTextAlignmentNatural;
```

Comment:
Before this was causing the text to appear on the left even for RTL languages
Now it appears on the left for LTR and the right for RTL